components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkConstants.java

```diff
@@ -356,6 +356,7 @@ public static class RequestParams {
         public static final String RESTART_FLOW = "restart_flow";
         public static final String LOGIN_TENANT_DOMAIN = "t";
         public static final String USER_TENANT_DOMAIN_HINT = "ut";
+        public static final String CORRELATION_ID = "correlationId";
```

Comment:
can we have a shorter url param? :)

Reply:
how about `traceId`?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/4347/commits/0bc0fbfb8c0890f484d45924e3e5bd64a203ea27